src/main/java/frc/robot/Constants.java

```diff
@@ -88,25 +99,22 @@
 	public final static int kSlot_Distanc = SLOT_0;
 	public final static int kSlot_Turning = SLOT_1;
 	public final static int kSlot_Velocit = SLOT_2;
-  public final static int kSlot_MotProf = SLOT_3;
-  //smartdashboard editible values
-  public final static String kAmpLimitStr = "Amp Limit";
```

Comment:
request was made to have AMP limit & Peak on screen so could test / change as needed